android/assets/jsons/translations/template.properties

```diff
@@ -1422,6 +1422,7 @@ Pillaging this improvement yields [stats] =
 Pillaging this improvement yields approximately [stats] = 
 Needs removal of terrain features to be built = 
 Unit type = 
+Toggle UI (World Screen only) =
```

Comment:
the check failed because this line needs a space after `=` :)

Reply:
That's funny. I know about the trailing space requirement and thought I put it in. =P